app/src/main/java/fr/free/nrw/commons/review/ReviewActivity.java

```diff
@@ -226,4 +230,10 @@ public boolean onOptionsItemSelected(MenuItem item) {
         }
         return super.onOptionsItemSelected(item);
     }
+
+    public ReviewImageFragment getInstanceOfReviewImageFragment(){
+        int pos=reviewPager.getCurrentItem();
```

Comment:
In Java we prefer to use full words for variable names, unlike other languages like C.